pkg/api/auth_middleware.go

```diff
@@ -195,11 +195,16 @@ func checkSecurityRequirements(r *http.Request,
 }
 
 func enhanceWithFriendlyName(ctx context.Context, user *model.User, friendlyName string, persistFriendlyName bool, authService auth.Service, logger logging.Logger) *model.User {
+	log := logger.WithFields(logging.Fields{"friendly_name": friendlyName, "persit_friendly_name": persistFriendlyName})
+	if user == nil {
+		log.Warning("user is nil")
```

Comment:
Do we know when this happens?  Might be worth saying, _even in the log_.

Also, if we don't know, this might flood the logs.  Probably still OK, but worth noting.

Reply:
I actually considered leaving this as a panic
Currently, this should never happen

Reply:
Then maybe log.Error?